### PR TITLE
disable task reconnection by default

### PR DESCRIPTION
**Pull request by meganrogge**

### CODE REVIEW

File: src/vs/workbench/contrib/tasks/common/tasks.ts [hunk 1]
```diff
@@ -1201,7 +1201,8 @@ export const enum TaskSettingId {
 	QuickOpenDetail = 'task.quickOpen.detail',
 	QuickOpenSkip = 'task.quickOpen.skip',
 	QuickOpenShowAll = 'task.quickOpen.showAll',
-	AllowAutomaticTasks = 'task.allowAutomaticTasks'
+	AllowAutomaticTasks = 'task.allowAutomaticTasks',
+	Reconnection = 'task.experimentalReconnection'
```
Comment (mjbvz) [inline src/vs/workbench/contrib/tasks/common/tasks.ts:1205]: For other experimental settings, we've been using `experimental` more like namespace, so `task.experimental.reconnection`. Does that make sense here too?

File: src/vs/workbench/contrib/tasks/browser/task.contribution.ts [hunk 2]
```diff
@@ -513,6 +513,11 @@ configurationRegistry.registerConfiguration({
 			description: nls.localize('task.showDecorations', "Shows decorations at points of interest in the terminal buffer such as the first problem found via a watch task. Note that this will only take effect for future tasks."),
 			default: true
 		},
+		[TaskSettingId.Reconnection]: {
+			type: 'boolean',
+			description: nls.localize('task.experimentalReconnection', "On window reload, reconnect to running watch/background tasks. Note that this is experimental, so you could encounter issues."),
```
Comment (mjbvz) [inline src/vs/workbench/contrib/tasks/browser/task.contribution.ts:518]: If it's experimental, it should also list `tags: ['experimental']`

**Review by mjbvz — COMMENTED** [review]
Just a minor question about the setting name

File: src/vs/workbench/contrib/tasks/common/tasks.ts (hunk 1 above)
Comment (meganrogge) [reply src/vs/workbench/contrib/tasks/common/tasks.ts:1205]: yep 👍🏼

**Review by mjbvz — APPROVED** [review]

**Pull request merged**